KenticoCloud.Delivery/KenticoCloud.Delivery.csproj

```diff
@@ -2,7 +2,6 @@
 
   <PropertyGroup>
     <TargetFrameworks>netstandard2.0;net45</TargetFrameworks>
-    <Version>5.0.0</Version>
```

Comment:
can you please check that version patching in appveyor works fine if you remove this?
how to verify:
- check that assembly & fileversions are set correctly (after a package is built in appveyor)
- check that sdk tracking header is set properly for outcoming http requests

Reply:
I have tried to download packages (artifacts) built by AppVeyor for v. 6.0.0-beta9 
* https://ci.appveyor.com/project/kentico/deliver-net-sdk/build/6.0.0-beta9

Tried to take a look to the dll properties:
* ![image](https://user-images.githubusercontent.com/9218736/45478408-274fcb00-b744-11e8-9d06-2694f7277d13.png)

I have tried to debug code uding **SourceLink** 😈 :
```
var client = new DeliveryClient("975bf280-fd91-488c-994c-2f04416e5ee3");
var request = client.GetItemAsync("on_roasts");
var result = request.Result;
```
And the version was correctly when sending the request:
* 
![image](https://user-images.githubusercontent.com/9218736/45478534-7433a180-b744-11e8-84c4-c07baaa937ea.png)

So I assume that Version tag is not necessary.

Reply:
ok